languages/go/exercises/concept/using-map/.docs/instructions.md

```diff
@@ -0,0 +1,22 @@
+In this excersie, you'll help uncle Donald to count monthly and total income of his thrift shop, he recorded of all of the transaction in his journal book, but he forgot to put the price it it, here's the price list
+
+| item name        | price |
+| ---------------- | ----- |
+| men_tops         | 2     |
+| men_underwares   | 1     |
+| men_jeans        | 4     |
+| robes            | 4     |
+| jackets          | 7     |
+| women_tops       | 4     |
+| women_underwares | 2     |
+| women_jeans      | 3     |
+
+## 1. Store the price table in your code, using map literal
```

Comment:
The task names usually describe what to do, less on how to do it. Maybe rephrase a bit to something like:

```suggestion
## 1. Define the price table
```